Objective-C/CBLDatabase.mm

```diff
@@ -1137,19 +1136,19 @@ - (void) postDatabaseChanged {
                     [_dbChangeNotifier postChange:
                         [[CBLDatabaseChange alloc] initWithDatabase: self
                                                         documentIDs: docIDs
-                                                         isExternal: external] ];
+                                                         isExternal: external]];
```

Comment:
This will be the previous external. once there are more than 1000 changes, or no more changes or new changes from different db, it will post to user. So external will be previous value. 

Once it's posted, it will create new dictionary to keep the new set of changes, and external value. 
I was also confused at beginning.

Reply:
I still don't understand the logic. How the first external (which is now always false)?